charts/cosigned/README.md

```diff
@@ -17,18 +17,14 @@ The previous command generates two key files `cosign.key` and `cosign.pub`. Next
 
 ```shell
 kubectl create namespace cosign-system
-
-kubectl create secret generic mysecret -n cosign-system --from-file=cosign.pub=./cosign.pub --from-file=cosign.key=./cosign.key --from-literal=cosign.password=$COSING_PASSWORD
 ```
 
 Install `cosigned` using Helm and setting the value of the secret key reference to `mysecret` that you created above:
 
 ```shell
 helm repo add sigstore https://sigstore.github.io/helm-charts
 
-helm repo update
```

Comment:
When adding a new repository to helm (line 27) helm will automatically pull the chart index from this repository. No need to perform a `helm repo update`.